### PR TITLE
Quick format change in subjectinfo.client.view.html

### PR DESCRIPTION
**Pull request by Blistin**
Rearranged the emergency info to be easier to read

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/3See/etect-app/pull/27" target="_blank">ReviewNinja

File: modules/core/client/views/subjectinfo.client.view.html [hunk 1]
```diff
@@ -48,12 +48,11 @@ <h1 class="text-center">{{info.fullName}}</h1>
               
           <!-- Display the contact info -->
           <p>Name: {{info.emergencyContact1.name}}</p>
-          <p>Name: {{info.emergencyContact2.name}}</p>
-
           <p> Relationship: {{info.emergencyContact1.relationship}}</p>
-          <p> Relationship: {{info.emergencyContact2.relationship}}</p>
-
           <p> Phone: {{info.emergencyContact1.phone}}</p>
+          </br>
```
Comment (Blistin) [inline modules/core/client/views/subjectinfo.client.view.html:53]: !fix Use margin css to fix vertical spacing, not br. Usually, br is done within text and paragraphs to move to the next line, not to put space between paragraphs themselves

Comment (naugle92) [inline modules/core/client/views/subjectinfo.client.view.html:53]: !fixed Removed the <br>s and added padding in the CSS

**Comment by Blistin** [comment]
Finished with #28 so forget about this one

**Pull request closed without merging**